fix: Revert native contains property

Alonski opened the pull request:
The change done in this PR: #466 caused a regression.

$.text() can find the text in `visibility: hidden` or `display: none` elements.

element.innerText does not return the text. 

Fixes: #514

ro0gr commented on addon-test-support/properties/contains.js:
```diff
@@ -1,6 +1,5 @@
-import { findMany, findOne } from '../extend';
-import { A } from '@ember/array';
+import { assign, every } from '../-private/helpers';
+import { findElementWithAssert } from '../extend';
```
internaly we tend to avoid using `findElementWithAssert(` in favour of `findMany` or `findOne`. I'd appreciate if we can avoid `findElementWithAssert` here as well.

ro0gr reviewed (commented):
Thanks for moving it forward! Mind adding a test case for this regression? Some unit test [here](https://github.com/Alonski/ember-cli-page-object/blob/master/tests/unit/-private/properties/contains-test.ts) should be fine.

ro0gr commented:
So turns out it doesn't really fix the https://github.com/san650/ember-cli-page-object/issues/514#issuecomment-942697666, which has never been a regression. But it fixes a real regression of the `contains(`. Going to squash the commits excluding the `Fixes: #514` part.

Alonski commented:
@ro0gr @san650 I believe the Ember community is moving towards Github Actions vs Travis in the future. So migrating to Actions isn't a bad idea 💡

Thanks for merging and releasing!

Alonski commented:
@ro0gr It seems that now I added a regression in this PR 🙃 

I added a dependency to the `every` helper function on jQuery which breaks in apps that don't have jQuery integration set to true.

I need to think of a way to fix this as we also can't use `element.text()` now.

Alonski commented:
@ro0gr Created a reproduction in StackBlitz:
https://stackblitz.com/edit/github-oujqsi?devtoolsheight=33&file=package.json

In console:
```sh
npm i
npm run test:ember --server
```

![image](https://user-images.githubusercontent.com/452199/137302820-83f72977-6cbe-4663-95d8-c67f99b16709.png)

Alonski commented:
@ro0gr Can you please also add the `hacktoberfest-accepted` label to this PR as well :)
